locale/de.yaml

```diff
@@ -0,0 +1,1848 @@
+try: "" #
+type: "" #
+val: "" #
+virtual: "" #
+when: "" #
+while: "" #
+with: "" #
+
+# << Error Messages >>
+
+# ----------------------
+# |   LANGUAGE: PHP    |
+# ----------------------
+# << Keywords >>
+__halt_compiler: ""
+abstract: ""
+and: ""
+array: ""
+as: ""
+break: ""
+callable: ""
+case: ""
+catch: ""
+class: ""
+clone: ""
+const: ""
+continue: ""
+declare: ""
+default: ""
+die: ""
+do: ""
+echo: ""
+else: ""
+elseif: ""
+empty: ""
+enddeclare: ""
+endfor: ""
+endforeach: ""
+endif: ""
+endswitch: ""
+endwhile: ""
+eval: ""
+exit: ""
+extends: ""
+final: ""
+for: ""
+foreach: ""
+function: ""
+global: ""
+goto: ""
+if: ""
+implements: ""
+include: ""
+include_once: ""
+instanceof: ""
+insteadof: ""
+interface: ""
+isset: ""
+list: ""
+namespace: ""
+new: ""
+or: ""
+print: ""
+private: ""
+protected: ""
+public: ""
+require: ""
+require_once: ""
+return: ""
+static: ""
+switch: ""
+throw: ""
+trait: ""
+try: ""
+unset: ""
+use: ""
+var: ""
+while: ""
+xor: ""
+__CLASS__: ""
+__DIR__: ""
+__FILE__: ""
+__FUNCTION__: ""
+__LINE__: ""
+__METHOD__: ""
+__NAMESPACE__: ""
+__TRAIT__: ""
+
+# << Error Messages >>
+
+# ----------------------
+# |  LANGUAGE: PYTHON  |
+# ----------------------
+# << Keywords >>
+#
+# translation syntax:
+#
+# # xxx? -> alternative word that may or may not fit better
+# # no idea -> cant think of a german equivalent
+# # works but weird -> its translated and makes sense, but sounds weird to me ...
+# ... needs second opinion on wether to translate or keep english
+
+# General comment to German translations; due to its grammatical structure,
+# German tends to use more than one word where only one is sufficient in
+# English. This means that many keywords are not easily translatable.
+
+  False: "Falsch" #✅✅✅✅✅✅
+  None: "Nichts" #✅✅✅✅✅✅
+  True: "Wahr" #✅✅✅✅✅✅
+  and: "und" #✅✅✅✅✅✅
+  as: "als" #✅✅✅✅✅✅
+  assert: "behaupte" # ✅ ? fordere #❓"bestätige"✅ or "versichere" might fit better ✅ (agree with "versichere")
+  async: "asynchron" #✅✅✅✅✅✅
+  await: "erwarte" #✅✅✅✅✅✅ #warte auf ?
+  # break: "bruch"
+  break: "brich" # Imperative instead of noun # Abbruch ? #❓❓"brich ab"✅✅ fits better; just "brich" rather means "to fracture something"
+  class: "klasse" #✅✅ #?Klasse is a noun # needs a capital "K" <- I second this one, the "K" needs to be uppercase <-- agree <- agree
+  continue: "fortfahren" #✅✅ # ? überspringe ✅ (imperative and the semantic meaning is clearer) # <- I second this # works but weird
+  def: "def" # <- short | definiere? ✅✅✅✅ #🤔
+  del: "lösche" #✅✅✅✅❓✅
+  # elif: "sonstwenn"✅
+  elif: "sonstfalls" #✅✅✅ # If -> falls, when -> wenn #ansonsten? ❓ "sonstfalls" sounds weird, would prefer "andernfalls"
+  else: "sonst" #✅✅✅✅✅✅
+  except: "außer" #✅✅✅✅✅✅
+  finally: "endlich" # ? schlußendlich ✅✅✅✅ # ? abschließend # I second "schlußendlich"
+  for: "für" #✅✅✅✅❓✅
+  from: "von" ✅✅ # ? aus ✅ (in the semantic context of python, this represents the meaning more clearly) # I second "aus" <-- agree
```

Comment:
I second "aus" as the usage would be
`aus ... importiere ...` 
which imho sounds better than
`von ... importiere ...`